daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -153,7 +153,7 @@ class KeysetValueTypeCalculatorOrdered(valueMap: HashMap[DataValuePrimitive, Dat
       })
       ans1 match {
         case None => {
-          (DataValue.NoValue, One(s"Key ${x} not found in keyset-value mapping"))
+          (DataValue.NoValue, One(s"Key ${x} not found in enumeration key-value mapping"))
```

Comment:
I would not even use the term "key" here. 

Parsing we're looking for the repValue of the enumeration.  So like "The value '993' was not found in any dfdlx:repValues for the enumeration type."

Unparsing we're looking for the enumeration value (to determine the corresponding repValue).  So like "The enumeration value 'xyzzy' was not defined."